src/faction_camp.cpp

```diff
@@ -1904,26 +1904,28 @@ void basecamp::scan_pseudo_items()
 
         tripoint mapmin = tripoint( 0, 0, omt_pos.z() );
         tripoint mapmax = tripoint( 2 * SEEX - 1, 2 * SEEY - 1, omt_pos.z() );
-        map &here = get_map();
+        get_map();
```

Comment:
This doesn't do anything.
```suggestion
```

Reply:
Hm, I thought it was another one of the C stupidities of functions with side effects, but looking at it it does indeed seem to just return data.

Thanks.